docs/content-services/components/alfresco-viewer.component.md

```diff
@@ -67,7 +67,8 @@ See the [Custom layout](#custom-layout) section for full details of all availabl
 | ---- | ---- | ------------- | ----------- |
 | allowDownload | `boolean` | true | Toggles downloading. |
 | allowFullScreen | `boolean` | true | Toggles the 'Full Screen' feature. |
-| allowGoBack | `boolean` | true | Allows `back` navigation |
+| allowGoBack | `boolean` | true | Allows `back` navigation. |
+| closeButtonPosition | `string` | `right` | Set close button position right/left. |
```

Comment:
@MichalKinas Lilian said to put right as default.
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/106949107/e3730926-8589-4db0-b167-6323016c8b57)

Reply:
That's okay for ACA, ADF is a different story since some people using this component right now, when migrating to latest ADF version in this case would have to override the default, additionally HX teams using this component would consider it a breaking change as they might not want to change the position, that's why the default value for ADF should stay as left

Reply:
ok got it, I will change default value for ADF as left